vanilla/addons
# Add separate content filter class to Civil Tongue Ex

This PR is a dependency for: https://github.com/vanillaforums/collegeconfidential/pull/362

**Steps to test:**
- In the [collegeconfidential](https://github.com/vanillaforums/collegeconfidential) repo checkout `fix/hobsons-civiltongueex` if it's not merged yet.
- Enable [Hobsons](https://github.com/vanillaforums/collegeconfidential/tree/master/plugins/Hobsons) plugin
- Enable CivilTongueEX
- Add some works in the blacklist and a replacement
- Make sure you have a discussion with some bad words in the title.
- Verify if the replacement displays properly and in the comment header section (between the comment and the discussion):
![Screen Shot 2020-03-04 at 12 07 12 PM](https://user-images.githubusercontent.com/31521020/75904184-bd802100-5e10-11ea-95f3-d862000760ad.png)

**Further testing:**
- Disable CivilTongueEX and make sure Hobsons doesn't crash for disabling a dependency.
